.github/workflows/upload-idf-component.yml

```diff
@@ -0,0 +1,20 @@
+name: Push components to https://components.espressif.com
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  upload_components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: "recursive"
+
+      - name: Upload components to the component registry
+        uses: espressif/github-actions/upload_components@master
+        with:
+          name: arduino-esp32
+          namespace: espressif
+          api_token: ${{ secrets.IDF_COMPONENT_API_TOKEN }}
+          service_url: https://api.staging.components.espressif.tools/
```

Comment:
Is this supposed to be replaced with production URL before merging?

Reply:
@magicarm22 Please remove the URL from here, if it's missing a default one will be used